packages/shopify-app-session-storage-mysql/src/mysql.ts

```diff
@@ -1,12 +1,27 @@
 import mysql from 'mysql2/promise';
 import {Session} from '@shopify/shopify-api';
-import {SessionStorage} from '@shopify/shopify-app-session-storage';
+import {
+  SessionStorage,
+  SessionStorageMigratorOptions,
+  SessionStorageMigrator,
+  RdbmsSessionStorageMigrator,
+  RdbmsSessionStorageOptions,
+} from '@shopify/shopify-app-session-storage';
+
+import {migrationMap} from './migrations';
+import {MySqlEngine} from './mysql-engine';
+
+export interface MySQLSessionStorageOptions
```

Comment:
This might break existing TS code since there are new mandatory fields in `RdbmsSessionStorageOptions`.